locales/sv_SE/pcsx2_Main.po

```diff
@@ -899,32 +899,30 @@ msgstr "Om %s"
 
 #: pcsx2/gui/Dialogs/AboutBoxDialog.cpp:54
 msgid "Website"
-msgstr ""
+msgstr "Webställe"
 
 #: pcsx2/gui/Dialogs/AboutBoxDialog.cpp:56
 msgid "Support Forums"
-msgstr ""
+msgstr "Stödforum"
 
 #: pcsx2/gui/Dialogs/AboutBoxDialog.cpp:58
 msgid "GitHub Repository"
-msgstr ""
+msgstr "GitHub's Magasin"
```

Comment:
While creative, I'm not sure about magasin. Perhaps just use 'repo' like any Swedish developer would.

Reply:
I'm not sure about "Magasin" eighter, but it's kinda fun, yes?